app/controllers/application_controller.rb

```diff
@@ -1,5 +1,19 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user = User.first
+  protect_from_forgery with: :exception
+  # before_action :authenticate_user!
```

Comment:
- [ ] I don't think your `devise` is necessary if you do not authenticate your user. Without `authenticate_user!` your `devise` won't be able to do it's job. For more information: what is [authenticate_user](https://www.google.com/search?q=what+is+authenticate_user+rails&rlz=1C5CHFA_enUZ1015UZ1015&sxsrf=ALiCzsZ8udM_E95KySVvzoqUGjdVaJf53g%3A1668184045400&ei=7XduY-L4F5rQqwH_5a-wBQ&oq=whatauthenticate_user+rails&gs_lcp=Cgxnd3Mtd2l6LXNlcnAQARgAMgoIIRDDBBAKEKABOgoIABBHENYEELADOg0IABDkAhDWBBCwAxgBSgQITRgBSgQIQRgASgQIRhgBUPgHWJARYK8caAFwAXgAgAG-AYgBnAWSAQMwLjSYAQCgAQHIAQ3AAQHaAQYIARABGAk&sclient=gws-wiz-serp#:~:text=Devise%20also%20comes%20with%20some%20very%20useful%20helper%20functions%3A%20before_action%20%3Aauthenticate_user!%20%E2%80%94%20Add%20to%20any%20controller%20to%20limit%20access%20to%20an%20action%20unless%20a%20user%20is%20logged%20in.) ?